### ISSUE TITLE
Adding timeout to requests

### ISSUE THREAD

**Issue by GregoirePelegrin** [JeanExtreme002/FlightRadarAPI#79]
**Is your feature request related to a problem? Please describe.**
- I am in a situation where connectivity issues may appear. I must then handle the case where the connection is lost at some point during the queries. When the connectivity is lost, the current API simply doesn't respond and blocks my code, not even throwing an error which could then be handled with a simple `try ... catch`.
- I would need the queries to fail properly instead of simply hanging when such connectivity issues happened.
- This would be easily solved by adding a timeout parameter to the queries as `requests` has this parameter.
This should be a parameter of the `APIRequest` class (maybe a default value, which could be overwritten for each instance?) and could also be a parameter of the `FlightRadar24API` class (as primary source of `APIRequest` instances)

**Comment by EugIva** [comment]
smth like this?
```
timing = time.time()
       ###
except requests.exceptions.ConnectionError or TimeoutError or requests.exceptions.ConnectTimeout \
               or AttributeError:
            self.out_filename = "1" + self.out_filename
            if timing < self.finish and len(self.out_filename) < 20:
                print("Connection error " + str(datetime.now()) + ". reconnectng...")
        
```

**Comment by JeanExtreme002** [comment]
Solved on PR #80
